### DISCUSSION TITLE
Bypass user login form

### DISCUSSION THREAD

**Discussion by nishit93-hub** [keycloak/keycloak#33781] [Q&A]
Hi,

I want to bypass the login form so the user can go directly to the registration page.
If i check the logic-action/authentication URL, keycloak generates tab_id and client_data. Is there a way to generate tab_id and client_data so that we can use details in our registration URL to bypass the login page

I also tried with URL 
`https://Oauth.ai/realms/clp/protocol/openid-connect/auth?client_id=clp_app&response_type=code&redirect_uri=https://localhost:8082/login/oauth2/code/oidc&kc_action=register` But still redirecting to login page.

**Comment by codespearhead** [comment]
That sounds like an XY problem [1].

We need to understand the actual problem you're trying to solve with your suggested solution.

***

[1] https://xyproblem.info/
